### PR TITLE
Flow ideas

### PR DESCRIPTION
**Pull request by Fermain**
Adds two demo files showing potential candidates for WYSIWYG and Diff diagrams. These could be used as a starting point to build a more tailored flow.

- [x] Installs Sveltekit using `Library` configuration in `packages/flow` with `npm create svelte@latest`
- [x] Adds DiffDemo svelte component leveraging `diff2html`
- [x] Adds EditorDemo svelte component leveraging `@toast-ui/editor` 

## Release Notes

- New Feature: Adds ESLint, Prettier, and Playwright configurations, as well as new Svelte components for displaying diffs and editors.
- Documentation: Updates README file and adds demo files.
- Test: Adds automated tests using Playwright.

> "Flowing with new tools and tests,
> This PR brings code quality to its best.
> With ESLint, Prettier, and Playwright too,
> Bugs beware, we're coming for you!"

### CODE REVIEW

File: packages/flow/svelte.config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import adapter from '@sveltejs/adapter-auto';
+import { vitePreprocess } from '@sveltejs/kit/vite';
```
[the remark this answers is not in the document]
Comment (Fermain) [reply packages/flow/svelte.config.js:2]: I will not!

File: packages/flow/src/lib/components/DiffDemo.svelte [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+	import * as Diff2Html from 'diff2html';
+	import 'diff2html/bundles/css/diff2html.min.css';
+
+	// An example of a pull request diff using Unified Diff format
```
[the remark this answers is not in the document]
Comment (Fermain) [reply packages/flow/src/lib/components/DiffDemo.svelte:5]: I am ok with this for now.

File: packages/flow/src/index.test.ts
Comment (pretzL) [inline packages/flow/src/index.test.ts:1]: Should definitely clean this out of the repo. Potentially now, definitely in the future

File: packages/flow/tests/test.ts
Comment (pretzL) [inline packages/flow/tests/test.ts:1]: This too

**Review by pretzL — APPROVED** [review]

**Review by Jenfer93 — APPROVED** [review]

**Pull request merged**